<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adjust</class>
 <widget class="QWidget" name="adjust">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frBg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>620</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frBg{
background-image:url(:/main/img/bg.jpg);
}
QFrame#frWhite{
background-color:rgb(255,255,255,230);
}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frWhite">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>850</width>
      <height>540</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	font-family:STXihei, &quot;华文细黑&quot;, &quot;Microsoft YaHei&quot;, &quot;微软雅黑&quot;;
font-size:20px;
}
QPushButton{
border-radius:15px;
background:rgb(237,200,199)

}
QPushButton:hover{
background:rgb(255,208,204);
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnDecode">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>450</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>解码</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPic">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>720</width>
       <height>405</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditThresh">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>450</y>
       <width>113</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>450</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>设定阈值</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnThreshComf">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>450</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>450</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>上一步</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>500</y>
       <width>401</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>STXihei,华文细黑,Microsoft YaHei,微软雅黑</family>
       <pointsize>-1</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:15px;
</string>
     </property>
     <property name="text">
      <string>注：阈值范围在0~255之间，保证视频整体完整即可</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>500</width>
       <height>40</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>221</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>正在解码，请等待...</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>621</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
